index.html

```diff
@@ -90,7 +116,20 @@ <h2 class="weekday"> Welcome! </h2>
         </div>
 
     </div>
+
+    <!-- Footer -->
+    <footer class="text-center py-3 mt-auto">
+        <div class="container">
+            <hr class="vertical-line">
+            <p>&copy; 2023 bszet.me<br> All rights reserved.<br>
+                <a href='/CHANGELOG.md' class="footer">Changelog</a>
+                <span>|</span>
+                <a href='https://youtu.be/xvFZjo5PgG0?si=dLyMSXAD7srWbOw8' class="footer">GitHub</a>
+            </p>
+        </div>
+    </footer>
```

Comment:
Can we tone down the color of the text down there, maybe to this (as this is the color of the <hr /> tag): #5c5d5e?

Reply:
definitely!